pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -50,38 +57,80 @@ func (r *REST) Destroy() {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	summary := obj.(*statsv1alpha1.NodeLatencyStats)
+	err := r.indexer.Update(summary)
+	if err != nil {
+		return nil, errors.NewInternalError(err)
+	}
+
+	return summary, nil
 }
 
 func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	obj, exists, err := r.indexer.GetByKey(name)
+	if err != nil || !exists {
+		return nil, errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), name)
+	}
+
+	return obj.(*statsv1alpha1.NodeLatencyStats), nil
 }
 
 func (r *REST) NewList() runtime.Object {
 	return &statsv1alpha1.NodeLatencyStats{}
 }
 
 func (r *REST) List(ctx context.Context, options *internalversion.ListOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStatsList{}, nil
+	objs := r.indexer.List()
+	if objs == nil {
+		return &statsv1alpha1.NodeLatencyStatsList{}, nil
+	}
+
+	entries := make([]statsv1alpha1.NodeLatencyStats, 0, len(objs))
+	for _, obj := range objs {
+		entries = append(entries, *obj.(*statsv1alpha1.NodeLatencyStats))
+	}
+
+	return &statsv1alpha1.NodeLatencyStatsList{Items: entries}, nil
 }
 
 func (r *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOptions runtime.Object) (*metav1.Table, error) {
```

Comment:
can you add an example of what this looks like when you call `kubectl get nodelatencystats` (add it as a PR comment or in the PR description)?

Reply:
Ok, I will append the usage.

Reply:
Example:

```bash
antrea git:(feat/impl-node-latency-query-api) ✗ kubectl get nodelatencystats
SOURCENODENAME       PEERNODELATENCYSTATS
kind-worker          [map[nodeName:kind-worker2 targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:1097048 lastRecvTime:2024-06-30T03:21:02Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.1.1]]] map[nodeName:kind-control-plane targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:1143333 lastRecvTime:2024-06-30T03:21:02Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.0.1]]]]
kind-worker2         [map[nodeName:kind-control-plane targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:571279 lastRecvTime:2024-06-30T03:21:12Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.0.1]]] map[nodeName:kind-worker targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:1257029 lastRecvTime:2024-06-30T03:21:02Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.2.1]]]]
kind-control-plane   [map[nodeName:kind-worker2 targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:1499582 lastRecvTime:2024-06-30T03:21:02Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.1.1]]] map[nodeName:kind-worker targetIPLatencyStats:[map[lastMeasuredRTTNanoseconds:1339598 lastRecvTime:2024-06-30T03:21:02Z lastSendTime:2024-06-30T03:21:12Z targetIP:10.244.2.1]]]]
```

Reply:
Thanks for the suggestion, I have converted the raw data into the three metrics you mentioned and returned it.